homeassistant/components/imeon_inverter/strings.json

```
@@ -135,25 +174,84 @@
+          "good_1": "System operating normally.",
+          "web_account": "Web account notification. Please refer to your monitoring panel.",
+          "info_soft": "Software information available. Please refer to your monitoring panel.",
+          "info_ond": "Inverter information available. Please refer to your monitoring panel.",
+          "info_bat": "Battery information available. Please refer to your monitoring panel.",
+          "info_smartlo": "Smart load information available. Please refer to your monitoring panel."
+        },
+        "state_attributes": {
+          "options": {
+            "state": {
+              "com_lost": "[%key:component::imeon_inverter::entity::sensor::timeline_type_msg::state::com_lost%]",
+              "warning_grid": "[%key:component::imeon_inverter::entity::sensor::timeline_type_msg::state::warning_grid%]",
+              "warning_pv": "[%key:component::imeon_inverter::entity::sensor::timeline_type_msg::state::warning_pv%]",
+              "warning_bat": "[%key:component::imeon_inverter::entity::sensor::timeline_type_msg::state::warning_bat%]",
+              "error_ond": "[%key:component::imeon_inverter::entity::sensor::timeline_type_msg::state::error_ond%]",
+              "error_soft": "[%key:component::imeon_inverter::entity::sensor::timeline_type_msg::state::error_soft%]",
+              "error_pv": "[%key:component::imeon_inverter::entity::sensor::timeline_type_msg::state::error_pv%]",
+              "error_grid": "[%key:component::imeon_inverter::entity::sensor::timeline_type_msg::state::error_grid%]",
+              "error_bat": "[%key:component::imeon_inverter::entity::sensor::timeline_type_msg::state::error_bat%]",
+              "good_1": "[%key:component::binary_sensor::entity_component::problem::state::off%]",
+              "web_account": "[%key:component::imeon_inverter::entity::sensor::timeline_type_msg::state::web_account%]",
+              "info_soft": "[%key:component::imeon_inverter::entity::sensor::timeline_type_msg::state::info_soft%]",
+              "info_ond": "[%key:component::imeon_inverter::entity::sensor::timeline_type_msg::state::info_ond%]",
+              "info_bat": "[%key:component::imeon_inverter::entity::sensor::timeline_type_msg::state::info_bat%]",
+              "info_smartlo": "[%key:component::imeon_inverter::entity::sensor::timeline_type_msg::state::info_smartlo%]"
+            }
+          }
+        }
+      },
+      "energy_pv": {
+        "name": "Today PV energy"
+      },
+      "energy_grid_injected": {
+        "name": "Today grid injected energy"
+      },
+      "energy_grid_consumed": {
+        "name": "Today grid consumed energy"
+      },
+      "energy_building_consumption": {
+        "name": "Today building consumption"
+      },
+      "energy_battery_stored": {
+        "name": "Today battery stored energy"
+      },
+      "energy_battery_consumed": {
+        "name": "Today battery consumed energy"
```

Those compound adjectives need a hyphen:

```suggestion
      "energy_grid_injected": {
        "name": "Today grid-injected energy"
      },
      "energy_grid_consumed": {
        "name": "Today grid-consumed energy"
      },
      "energy_building_consumption": {
        "name": "Today building consumption"
      },
      "energy_battery_stored": {
        "name": "Today battery-stored energy"
      },
      "energy_battery_consumed": {
        "name": "Today battery-consumed energy"
```